lint-staged.config.js

```diff
@@ -1,13 +0,0 @@
-// lint-staged.config.js
-const micromatch = require("micromatch");
-
-module.exports = {
-  "*.js": files => {
-    // from `files` filter those _NOT_ matching `*test.js`
-    const match = micromatch.not(
-      files,
-      "elements/**/*.*.js **/analysis-error.json themes/**/*.*.js libraries/**/*.js"
-    );
-    return `eslint ${match.join(" ")}`;
```

Comment:
this appears to be left over?
ESLint doesn't appear to have a configuration in this project